<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aboba</class>
 <widget class="QWidget" name="aboba">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>537</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QPushButton" name="LoadAboba">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="MLPType">
      <item>
       <property name="text">
        <string>Matrix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Graph</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="AF">
      <item>
       <property name="text">
        <string>Sigmoid</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tanh</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ReLU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Leaky ReLU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ELU</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="DeleteAboba">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="CreateAboba">
      <property name="text">
       <string>Add New</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MLP type</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Activation Function</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QListWidget" name="AbobaView"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Topology</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Topology"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ErrorWindow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
